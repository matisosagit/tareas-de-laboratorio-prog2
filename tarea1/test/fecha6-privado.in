crearFecha 20/2/1988
imprimirFecha
aumentarDias 11
imprimirFecha
liberarFecha

# año bisiesto
crearFecha 1/2/2024
imprimirFecha
aumentarDias 28
imprimirFecha
aumentarDias 560
imprimirFecha
liberarFecha

# mas de un mes
crearFecha 18/12/2023
imprimirFecha
aumentarDias 65
imprimirFecha
liberarFecha

# mas de un mes y bisiesto
crearFecha 18/12/2023
imprimirFecha
aumentarDias 73
imprimirFecha
liberarFecha

# mas de un año
crearFecha 18/12/2022
imprimirFecha
aumentarDias 366
imprimirFecha
aumentarDias 377
imprimirFecha
liberarFecha

# mas de un año y bisiesto
crearFecha 18/12/2023
imprimirFecha
aumentarDias 368
imprimirFecha
liberarFecha
Fin
