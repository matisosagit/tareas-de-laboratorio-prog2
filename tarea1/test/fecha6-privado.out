1>2>20/2/1988
3>La nueva fecha aplazada 11 dias es: 2/3/1988
4>2/3/1988
5>6>#  año bisiesto.
7>8>1/2/2024
9>La nueva fecha aplazada 28 dias es: 29/2/2024
10>29/2/2024
11>La nueva fecha aplazada 560 dias es: 11/9/2025
12>11/9/2025
13>14>#  mas de un mes.
15>16>18/12/2023
17>La nueva fecha aplazada 65 dias es: 21/2/2024
18>21/2/2024
19>20>#  mas de un mes y bisiesto.
21>22>18/12/2023
23>La nueva fecha aplazada 73 dias es: 29/2/2024
24>29/2/2024
25>26>#  mas de un año.
27>28>18/12/2022
29>La nueva fecha aplazada 366 dias es: 19/12/2023
30>19/12/2023
31>La nueva fecha aplazada 377 dias es: 30/12/2024
32>30/12/2024
33>34>#  mas de un año y bisiesto.
35>36>18/12/2023
37>La nueva fecha aplazada 368 dias es: 20/12/2024
38>20/12/2024
39>40>Fin.
